source/Octopus.DotNet.Cli/Octopus.DotNet.Cli.csproj

```diff
@@ -6,7 +6,7 @@
     We include all possible targets here so that no matter what framework version
     the user has installed, there is an appropriate build in the tools package
     -->
-    <TargetFrameworks>netcoreapp2.1;netcoreapp2.2;netcoreapp3.0;netcoreapp3.1;net5.0</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.1;net5.0</TargetFrameworks>
```

Comment:
Should this be `net5.0` only?

Reply:
Nah, this is for the dotnet tool - we list both supported dotnet versions here